### PR TITLE
New: TreePickerSimple Component

### PR DESCRIPTION
**Pull request by camjc**
- Update package.json
- Text cleanups.
- Update value to be optional in TreePickerNodes.
- New: Export BorderedWell and PageTitle from Alexandria.
screen shot 2016-03-15 at 12 57 13 pm

### CODE REVIEW

File: src/components/adslotUi/fastStatelessWrapper.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import _ from 'lodash';
...
 
 /**
-* fastStatelessWrapper - Limit re-render of a component to certain changeable props.
-*                        Use when you have found a performance problem, do not prematurely optimize.
+* fastStatelessWrapper - Limit re-render of a component to changeable props.
+*                        Use when you have a performance problem, do not prematurely optimise.
 *
 * @param  {React Component} ComposedComponent The component to render.
-* @return {Array}           propsToCheck      An Array of properties that can cause re-render eg. `['foo.bar', 'baz']`.
+* @return {Array}           propsToCheck      An Array of properties causing re-render eg. `['foo.bar', 'baz']`.
```
Comment (OndrejCholeva) [inline src/components/adslotUi/fastStatelessWrapper.js]: (while touching this) Can you change `@return {Array}` to `@param {Array}` and add descriptions of whatever it returns?

Comment (camjc) [inline src/components/adslotUi/fastStatelessWrapper.js]: Ah, oops! Definitely.

File: src/components/Main.js [hunk 2]
```diff
@@ -308,7 +312,7 @@ class AppComponent extends React.Component {
         <Select
           clearable={false}
           name="countriesSelect"
-          noResultsText="Sorry, couldn't find that country."
+          noResultsText="No matching countries."
```
Comment (antklim) [inline src/components/Main.js]: Is it possible to move "No matching countries." to constants?

Comment (camjc) [inline src/components/Main.js]: This is just for the examples page mate, I'd rather leave it obvious.

File: test/components/adslotUi/TreePickerGridComponentTest.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
+import React from 'react';
+import TreePickerGrid from 'components/adslotUi/TreePickerGridComponent.js';
```
Comment (antklim) [inline test/components/adslotUi/TreePickerGridComponentTest.js]: `.js` :(

File: test/components/adslotUi/TreePickerSimplePureComponentTest.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import React from 'react';
+import SplitPaneComponent from 'components/adslotUi/SplitPaneComponent';
+import TreePickerMocks from 'mocks/TreePickerMocks';
+import TreePickerNav from 'components/adslotUi/TreePickerNavComponent.js';
```
Comment (antklim) [inline test/components/adslotUi/TreePickerSimplePureComponentTest.js]: `.js` :((

File: src/components/adslotUi/TreePickerGridComponent.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+import fastStatelessWrapper from 'components/adslotUi/fastStatelessWrapper';
+import TreePickerNode from 'components/adslotUi/TreePickerNodeComponent';
+import TreePickerPropTypes from 'helpers/propTypes/TreePickerPropTypes';
+import React, { PropTypes } from 'react';
```
Comment (antklim) [inline src/components/adslotUi/TreePickerGridComponent.js]: Should we follow `direct-web` conventions and import global dependencies prior locals?

File: src/components/Main.js (hunk 2 above)
Comment (antklim) [inline src/components/Main.js]: I understand, but it used twice :)

File: test/components/adslotUi/TreePickerGridComponentTest.js (hunk 3 above)
Comment (camjc) [inline test/components/adslotUi/TreePickerGridComponentTest.js]: haha. You should add this to the pre-commit hook mate.

File: src/components/adslotUi/TreePickerGridComponent.js (hunk 5 above)
Comment (camjc) [inline src/components/adslotUi/TreePickerGridComponent.js]: deps are imported as
import foo from 'foo'
then
import foo, {bar} from 'foo'
then
import {bar} from 'foo'

alphabetically in each list.

**Comment by vinteo** [comment]
LGTM

**Comment by vinteo** [comment]
Merging...

**Pull request merged**